src/main/resources/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep/config.groovy

```diff
@@ -46,8 +47,28 @@ f.section(title: descriptor.displayName) {
     f.select()
   }
 
-  f.entry(title: _(Messages.IqPolicyEvaluation_Application()), field: 'iqApplication') {
-    f.select()
+  //Instance is null if this is a new Item.  We need to set a default.
+  String manualAppId = ''
```

Comment:
I think you can use the groovy elvis operator here.  http://groovy-lang.org/operators.html#_elvis_operator